src/ethereum/base_types.py

```diff
@@ -55,15 +55,13 @@ def from_be_bytes(cls: Type, buffer: "Bytes") -> "Uint":
     def from_le_bytes(cls: Type, buffer: "Bytes") -> "Uint":
         return cls(int.from_bytes(buffer, "little"))
 
-    def __new__(cls: Type, value: int) -> "Uint":
+    def __init__(self, value: int) -> None:
```

Comment:
Ah. I guess since we don't modify the int, we don't need the whole `__new__` nonsense. Nice!